### ISSUE TITLE
Could not find the 'mermaid' executable in PATH

### ISSUE THREAD

**Issue by eustimenko** [asciidoctor/asciidoctor-gradle-plugin#446]
Have `build.gradle.kts` configuration

```kotlin
tasks {
    named<AsciidoctorTask>("asciidoctor") {
        requires("asciidoctor-diagram/mermaid")
        sourceDir = file("docs")
        sources(delegateClosureOf<PatternSet> {
            include("service-diagram.adoc")
        })
        outputDir = file("build/docs")
    }
}
```

and `service-diagram.adoc`

`[mermaid, service-flowchart, png]
....
graph LR
    Start --> Stop
....`

When `./gradlew asciidoctor` is launched there is an error:
`asciidoctor-diagram: ERROR: Failed to generate image: Could not find the 'mermaid' executable in PATH; add it to the PATH or specify its location using the 'mermaid' document attribute`

But I don't like to install `mermaid`. Any ideas?

**Comment by ysb33r** [comment]
@eustimenko Which version of the plugin?

**Comment by ysb33r** [comment]
Also IIRC, the Mermaid CLI needs to be installed as the GEM only provides a wrapper. Maybe @pepijnve can provide some context.

**Comment by pepijnve** [comment]
@ysb33r that's correct. asciidoctor-diagram is just a glue layer between asciidoctor and various diagram generating tools. An error like the one above most likely means mermaid is not installed on the system. It could also mean that the directory containing the `mermaid` executable is not in `PATH`. The asciidoctor-diagram docs explain how to specify the location of the various tools explicitly via document attributes.

**Comment by ysb33r** [comment]
I have something in mind to fix this. There are a bunch of asciidoctor-diagram cases that sufffers the same fate.

**Comment by pepijnve** [comment]
asciidoctor-diagram has rendering server support as well. If people prefer to do so they can delegate to, for instance, kroki.io.
